app/src/main/res/layout/update_organizer_layout.xml

```diff
@@ -59,22 +61,23 @@
                     android:layout_height="@dimen/submit_button_width"
                     android:text="@string/done"/>
 
-                <FrameLayout
-                    android:layout_width="@dimen/progressbar_large"
-                    android:layout_height="@dimen/progressbar_large"
-                    android:layout_gravity="center">
+            </LinearLayout>
+        </LinearLayout>
```

Comment:
Why's this needed?

Reply:
When layout was inside the linear layout, as when the progress bar will get its visibility being at end of screen it remain out of the screen. So I moved the Frame Layout of Progress bar as when it is visible it come above the linear layout.

Reply:
Add margin-bottom of button width in the nested scroll view as you can see in different edit layout's present in project structure.

Reply:
defining a margin bottom can create problem in future as if design or height of button is changed then we will also require to change the margin accordingly